scripts/load-backup.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/sh
+
+set -e
+
+# Check arguments
+if [ "$#" -ne 1 ]; then
+  echo "usage: $0 osrd.backup"
+  exit 1
+fi
+
+# Check sha1 is matching
+echo Checking backup integrity...
+BACKUP_PATH="$1"
+BACKUP_FILENAME=$(basename -s ".backup" "$BACKUP_PATH")
+EXPECTED_SHA1=$(grep -o -E '[0-9a-f]{40}' <<< $(echo "$BACKUP_FILENAME") || echo "renamed")
+CURRENT_SHA1=$(cut -d' ' -f1 <<< $(sha1sum "$BACKUP_PATH"))
+if [ "$EXPECTED_SHA1" == "$CURRENT_SHA1" ]; then
+  echo "  ✔ The backup is valid"
+elif [ "$EXPECTED_SHA1" == "renamed" ]; then
+  echo "  ✘ Invalid backup file name (you may have renamed the file)"
+  echo "    Found:    '${BACKUP_FILENAME}'"
+  echo "    Expected: 'osrd_full_01_01_2000_sha1_0000000000000000000000000000000000000000.backup' "
+  echo "               └───┬───┘ └───┬────┘ └─────────────────────┬─────────────────────┘ └─┬──┘  "
+  echo "                 name      date                   sha1 of the backup             extension"
+  exit 2
+else
+  echo "  ✘ Corrupted backup (you should download it again)"
+  echo "    Expected sha1: ${EXPECTED_SHA1}"
+  echo "    Current sha1:  ${CURRENT_SHA1}"
+  exit 2
+fi
+
+# Check if the database exists
+echo "Checking database exists..."
+DB_EXISTS=$(docker exec osrd-postgres psql -c "SELECT EXISTS (SELECT FROM pg_stat_database WHERE datname = 'osrd');")
+DB_EXISTS=$(grep -o -E '[tf]$' <<< $(echo "$DB_EXISTS"))
+
+if [ $DB_EXISTS == 't' ]; then
+  echo "  Database 'osrd' found"
+else
+  echo "  Database 'osrd' not found"
+fi
+
+if [ $DB_EXISTS == 't' ]; then
+  # Check that no service is connected to the database
+  echo "Checking database availability..."
+  DB_CONN=$(docker exec osrd-postgres psql -c "SELECT numbackends FROM pg_stat_database WHERE datname = 'osrd';")
+  DB_CONN=$(grep -o -E '[0-9]+$' <<< $(echo "$DB_CONN"))
```

Comment:
There are probably some quotes missing around here

```suggestion
  DB_CONN="$(docker exec osrd-postgres psql -c "SELECT numbackends FROM pg_stat_database WHERE datname = 'osrd';")"
  DB_CONN="$(echo "$DB_CONN" | grep -o -E '[0-9]+$')"
```